nilearn/plotting/html_stat_map.py

```diff
@@ -0,0 +1,97 @@
+import os
+import tempfile
+import base64
+
+import numpy as np
+from nilearn import image, datasets
```

Comment:
Relative imports would be better. Also, they should be grouped with the imports below (no empty line below) and with an empty line above: grouping local imports together.